safed.dd

```diff
@@ -2,56 +2,58 @@ Ddoc
 
 $(D_S SafeD&mdash;The Safe Subset of D,
 
+<img src="images/dman-rain.jpg" border=0 alt="D-man in rain" align=right alt="D Style" height=200>
+
 	$(P
 	$(SMALL by Bartosz Milewski, a member of the D design team)
 	)
 
 	$(P
-	I've seen some very good programmers move away from C++ in favor of languages like Java or C#. Being a hard-core C++ programmer myself, I wondered why anyone would want to switch to a less powerful and less efficient language. Mind you, I could understand why a newcomer would opt for a simpler, flatter-learning-curve language but, once somebody invested the time and effort to become proficient in C++, why in the world would they want to abandon it? 
+	I've seen some very good programmers move away from C++ in favor of languages like Java or C#. Being a hard-core C++ programmer myself, I wondered why anyone would want to switch to a less powerful and less efficient language. Mind you, I could understand why a newcomer would opt for a simpler, flatter-learning-curve language but, once somebody invested the time and effort to become proficient in C++, why in the world would they want to abandon it?
```

Comment:
seems like my VIM automatically removed the trailing whitespace. I guess that's a relict from old times. Should I remove this from this the PR?